lib/v2/gzh360/gzh.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { mpIdEncode, mpIdDecode, finishArticleItem } = require('./utils');
+const config = require('@/config').value;
+
+module.exports = async (ctx) => {
+    let name = ctx.params.name;
+    let id = name;
+
+    // in order to support RSSHub Radar, we need to accept both name and id
+    try {
+        name = mpIdDecode(name);
+    } catch {
+        id = mpIdEncode(name);
+    }
+
+    const rootUrl = 'http://web.gzh360.com';
+    const currentUrl = `${rootUrl}/gzh?id=${id}`;
+
+    const respData = await ctx.cache.tryGet(
+        currentUrl,
+        async () =>
+            await got(currentUrl)
+                .then((_r) => _r.data)
+                .catch((err) => {
+                    if (err.response.statusCode === 404) {
+                        throw new Error('This WeChat Official Account is not found on GZH360.');
+                    }
+                    throw err;
+                }),
+        config.cache.routeExpire,
+        false
+    );
+
+    const $ = cheerio.load(respData);
+
+    let items = $('div.newslist5 > li')
+        .map((_, item) => {
+            item = $(item);
+            const link = item.find('a');
```

Comment:
Combined with `/gzh_articles`
```suggestion
    let items = $('.content .image')
        .map((_, item) => {
            item = $(item);
            const link = item.find('a').first();
```